bridge/svix-bridge/src/main.rs

```diff
@@ -315,7 +315,10 @@ async fn main() -> Result<()> {
             "Starting JS Transformation Workers: {}",
             cfg.transformation_worker_count
         );
-        let pooler = runtime::JsPooler::new(cfg.transformation_worker_count);
+
+        deno_core::JsRuntime::init_platform(None, false);
```

Comment:
I remember at one point we had this and then we didn't (like maybe it was invoked by something in deno) and there was something about it causing a segfault if not run _exactly once_.

Just wanted to mention in case there's a better spot in the program to do this. `main()` seems fine to me. Perhaps it could also go inside JsPooler::new() in a once cell or whatever?

Reply:
I'm just going to leave it here for now.